tools/licenses/README

```diff
@@ -1,8 +1,5 @@
-To update the sky_engine LICENSE file, run the following in this directory:
+To update the sky_engine LICENSE file, run the following from the build root:
 
 ```
-pub get
-gclient sync
-dart lib/main.dart ../../.. > ../../sky/packages/sky_engine/LICENSE
-dart --checked lib/main.dart ../../.. > ../../travis/licenses.golden
+./flutter/tools/licenses/update_licenses.sh
```

Comment:
How does the diff indicate that? I just pasted the script we used to update the files for my last PR. Let me try adding another file and see what happens.